account-kit/react/src/tailwind/components/input.ts

```diff
@@ -4,7 +4,6 @@ export const inputComponents: ComponentDef = {
   ".input": {
     // container styling
     "@apply p-3 inline-flex gap-2 h-10 items-center": {},
-    "@apply transition-colors": {},
```

Comment:
this was behaving funny when switching between light and dark modes and wasn't adding much anyway